components/cli/cmd/cellery/test.go

```diff
@@ -74,11 +78,22 @@ func newTestCommand(cli cli.Cli) *cobra.Command {
 						"[<instance>:]<key>=<value>, received %s", envVar)
 				}
 			}
+
+			if debug {
+				if projLocation == "" {
+					return fmt.Errorf("expects a Ballerina project. Use --project-location or -p to specify the project location")
+				} else {
+					isExists, err := util.FileExists(filepath.Join(projLocation, constants.BallerinaToml))
+					if err != nil || !isExists {
+						return fmt.Errorf("expects a Ballerina project location, recieved %s", projLocation)
```

Comment:
The error does not get propagated to top if err != nil.

Reply:
fixed